README.md

```diff
@@ -2,11 +2,50 @@
 
 ## Stage 1
 
-This is a [proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
+This is a [proposal](https://tc39.github.io/process-document/), the result of a merge between an earlier draft of itself and Christophe Porteneuve's [proposal-numeric-underscores](https://github.com/tdd/proposal-numeric-underscores), extends the existing [_NumericLiteral_](https://tc39.github.io/ecma262/#prod-NumericLiteral) to allow a separator character between digits.
+
 
 ## Motivation
 
-This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. 
+This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. Large numeric literals are difficult for the human eye to parse quickly, especially when there are long digit repetitions. This impairs both the ability to get the correct value / order of magnitude...
```

Comment:
Can we merge this and then pare it down in a follow up? I ask because I'm not sure what should be removed because I think all of the examples are useful in their own way. @tdd's section is good for introducing the issue and showing the solution. My examples are more like "here are real world use cases, that you've probably encountered, and here's how this proposed feature improves on each".

So yeah... I don't know?